packages/studio-base/src/players/RosbridgePlayer.ts

```diff
@@ -45,48 +45,34 @@ const log = Log.getLogger(__dirname);
 
 const CAPABILITIES = [PlayerCapabilities.advertise, PlayerCapabilities.callServices];
 
-type RosNodeDetails = Record<
-  "subscriptions" | "publications" | "services",
-  { node: string; values: string[] }
->;
-
-function collateNodeDetails(
-  details: RosNodeDetails[],
-  key: keyof RosNodeDetails,
-): Map<string, Set<string>> {
-  return _.transform(
-    details,
-    (acc, detail) => {
-      const { node, values } = detail[key];
-      for (const value of values) {
-        if (!acc.has(value)) {
-          acc.set(value, new Set());
-        }
-        acc.get(value)?.add(node);
-      }
-    },
-    new Map<string, Set<string>>(),
-  );
-}
-
```

Comment:
I'd prefer you commenting out the old parts instead of deleting them. In the future it would be easier to recover a functionality we may have removed for simplicity in this first integration

Reply:
Same jaja